packages/ckeditor5-package-generator/lib/utils/is-yarn-installed.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license Copyright (c) 2020-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { execSync } = require( 'child_process' );
+
+/**
+ * @returns {Boolean}
+ */
+module.exports = function isYarnInstalled() {
+	const result = execSync( 'yarn -v || echo false', { stdio: [ null, 'pipe', 'ihnerit' ] } );
```

Comment:
Typo `'ihnerit'`, but this is probably not needed anyway and `null` could be set for `stderr`.